From f4f1096175be8a34f16fb3e39dfde22e4566d0c6 Mon Sep 17 00:00:00 2001
From: Paolo Bonzini <pbonzini@redhat.com>
Date: Fri, 22 Feb 2013 17:36:34 +0100
Subject: [PATCH 232/246] migration: remove useless qemu_file_get_error check

migration_put_buffer is never called if there has been an error.

Reviewed-by: Orit Wasserman <owasserm@redhat.com>
Reviewed-by: Juan Quintela <quintela@redhat.com>
Signed-off-by: Paolo Bonzini <pbonzini@redhat.com>
Signed-off-by: Juan Quintela <quintela@redhat.com>
---
 migration.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/migration.c b/migration.c
index 35c3455..d75beca 100644
--- a/migration.c
+++ b/migration.c
@@ -523,12 +523,6 @@ static int migration_put_buffer(void *opaque, const uint8_t *buf,
 
     DPRINTF("putting %d bytes at %" PRId64 "\n", size, pos);
 
-    ret = qemu_file_get_error(s->file);
-    if (ret) {
-        DPRINTF("flush when error, bailing: %s\n", strerror(-ret));
-        return ret;
-    }
-
     if (size <= 0) {
         return size;
     }
-- 
1.8.2.1

